### DISCUSSION TITLE
samplers selection

### DISCUSSION THREAD

**Discussion by kalle07** [mcmonkeyprojects/SwarmUI#367] [Ideas]
first
is ot possible to hide specific samplers i never use ?

second
you can add
FORGE "FLUX realistic" sampler ?

**Comment by mcmonkey4eva** [comment]
Regarding Forge's "flux realistic", see here: https://github.com/lllyasviel/stable-diffusion-webui-forge/pull/2151

Which identifies that the sampler is in fact a malicious mislabeling of DPM++ 2S Ancestral, which you can find in Swarm here:
![image](https://github.com/user-attachments/assets/2587ca88-3348-49a7-91ae-8b13371dfc35)

**Comment by kalle07** [comment]
i see...

fand
is it possible to hide specific samplers i never use ?

**Reply by mcmonkey4eva** [reply]
Not currently no. Arguably that's a valid feature request though, so can post it under Issues
